cli/dcoscli/data/help/job.txt

```diff
@@ -0,0 +1,46 @@
+Description:
+    Deploy and manage jobs in DC/OS.
+
+    Usage:
+        dcos job [--help][--version][--config-schema][--info]
+        dcos job add <job-file>
+        dcos job remove <job-id> [--stopCurrentJobRuns]
+        dcos job show <job-id>
+        dcos job update <job-file>
+        dcos job kill <job-id> [run-id][--all]
+        dcos job run <job-id>
+        dcos job list [--json]
+        dcos job schedule add <job-id> <schedule-file>
+        dcos job schedule show <job-id> [--json]
+        dcos job schedule remove <job-id> <schedule-id>
+        dcos job schedule update <job-id> <schedule-file>
+        dcos job show runs <job-id> [--run-id <run-id>][--q]
```

Comment:
dcos job show -> shows the job details
dcos job runs show -> shows the current runs for that job.  runs with no run-id is all runs, with-run-id is just that run.
So this is correct.